rosetta-source/src/main/resources/result-json-files/fpml-5-10/products/rates/bond-fwd-generic-ex01.json

```diff
@@ -93,20 +93,6 @@
             "source" : "ISDA",
             "productQualifier" : "InterestRate_Forward_Debt"
           } ],
-          "productIdentifier" : [ {
-            "value" : {
-              "identifier" : {
-                "value" : "JP1700161P53",
```

Comment:
This `productIdentifier` mapping has been dropped and needs to be reinstated.

This is because in the refactoring of `ProductBase`, the `productIdentifier` attribute has been dropped and disappeared from `ContractualProduct`, therefore it can't be mapped. The `productIdentifier` attribute should be reinstated in `ContractualProduct` directly (to be further adjusted in P3)